client/jetpack-cloud/sections/partner-portal/primary/issue-license/index.tsx

```diff
@@ -7,7 +7,11 @@ import IssueLicenseForm from 'calypso/jetpack-cloud/sections/partner-portal/issu
 import SidebarNavigation from 'calypso/jetpack-cloud/sections/partner-portal/sidebar-navigation';
 import AssignLicenseStepProgress from '../../assign-license-step-progress';
 
-export default function IssueLicense(): ReactElement {
+interface Props {
+	site_id?: number;
+}
+
+export default function IssueLicense( { site_id }: Props ): ReactElement {
```

Comment:
Can we rename this prop to `siteId`? It is fine to use `site_id` for a query parameter, but it is better to use camel case when we pass it down to a component.